README.md

```diff
@@ -42,6 +42,40 @@ A tool to automate front-end development tasks and streamline prototyping.
   * `http://[project].local:8000` if you're running Butler on a Vagrant
   * [http://localhost:8000](http://localhost:8000) if you're developing locally
 
+## Add Butler to a project, with spress
+
+1. Log in to your vagrant environment and navigate to your project. You should always run `npm` commands from within your Vagrant.
+1. Create a `styleguide` directory in your project
+1. **You'll install the styleguide dependencies and run Butler from the `styleguide` directory.** Go there now: `cd styleguide`
+1. Run `npm init` to create your `package.json` file
+1. Add Butler as a dependency:
+
+  ```
+npm install --save-exact palantirnet/butler#spress-task
+```
+  * Butler will prompt you for the git repository URL; respondez-vous
+  * Butler will ask you whether you're using Spress; say 'y'
+1. Add Butler's scripts to your `package.json` file:
+
+  ```json
+"scripts": {
+  "butler": "node_modules/butler/node_modules/.bin/gulp --gulpfile node_modules/butler/Gulpfile.js",
+  "develop": "node_modules/butler/node_modules/.bin/gulp --gulpfile node_modules/butler/Gulpfile.js develop",
+  "tests": "node_modules/butler/node_modules/.bin/gulp --gulpfile node_modules/butler/Gulpfile.js test",
+  "deploy": "node_modules/butler/node_modules/.bin/gulp --gulpfile node_modules/butler/Gulpfile.js deploy"
+},
+```
+1. Copy the contents of Butler's `STYLEGUIDE_TEMPLATE_SPRESS` directory into your `styleguide` directory with the following two commands:
+
+  ```
+cp -r node_modules/butler/STYLEGUIDE_TEMPLATE_SPRESS/* .
+cp -r node_modules/butler/STYLEGUIDE_TEMPLATE_SPRESS/.gitignore .
```

Comment:
Gah! It looks like `npm` strips the .gitignore files from the copy of butler! wtf.